notebooks/122-pytorch-post_training_quantization_and_sparsity_pot/122-pytorch-post-training-quantization-sparsity-pot-public.ipynb

```diff
@@ -0,0 +1,668 @@
+{
```

Comment:
<p>does notebook should somehow mention that prepared model can be run on NPU?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>